### ISSUE TITLE
Bug: Chrome security settings (default) doesn't allow extension to work properly

### ISSUE THREAD

**Issue by mordaha** [krasimir/kuker#3]
By default chrome blocks 3rd party cookies/websites

(Settings -> Content Settings -> Cookies
```
Block third-party cookies
Prevent third-party websites from saving and reading cookie data - On
```

This also blocks kuker access to localStorage, it fails to start and trying to switch to websocket-mode, and shows blank screen on Kuker tab

```
kuker-ui.js:43621 Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at getFilterTypes (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:43621:15)
    at initialState (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:43665:18)
    at Object.517.../constants (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:43673:10)
    at s (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:1:254)
    at chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:1:305
    at Object.515../components/App.jsx (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:43382:17)
    at s (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:1:254)
    at e (chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:1:425)
    at chrome-extension://glgnienmpgmfpkigngkmieconbnkmlcn/kuker-ui.js:1:443
```

As this security setting is essential and enabled by default, it should not block kuker extension

**Comment by krasimir** [comment]
Very good point. I hope to fix it quickly.

**Comment by krasimir** [comment]
Fixed in `5.4.4` version. Because the localStorage is not available the filter of events is not persistent.
